mistralrs-core/src/sampler.rs

```diff
@@ -35,6 +35,23 @@ pub struct SamplingParams {
     pub n_choices: usize,
 }
 
+impl Default for SamplingParams {
+    fn default() -> Self {
+        Self {
+            temperature: None,
+            top_k: None,
+            top_p: None,
+            top_n_logprobs: 0,
+            frequency_penalty: None,
+            presence_penalty: None,
+            stop_toks: None,
+            max_len: None,
+            logits_bias: None,
+            n_choices: 1,
```

Comment:
it was defaulting to 0

Reply:
I'm not sure if this should use the Completion Request `best_of` instead. These 2 sources of truth are confusing

Reply:
@lucasavila00 the `best_of` parameter sets the number of seqs to consider when they are done by cumulative logprob. It should be <= `n_choices`.